### PR TITLE
Finished Project 2

### PR DESCRIPTION
**Pull request by martin-mueller-xyz**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -63,12 +66,49 @@ send me a DM to check your pull request
...
+    ignoreUnused(wheel);
+    ignoreUnused(rolling);
+    ignoreUnused(sync);
+    ignoreUnused(start);
+    ignoreUnused(frequence);
+    ignoreUnused(resonance);
+    ignoreUnused(modulation);
+    ignoreUnused(chorus);
+    ignoreUnused(flanger);
+    ignoreUnused(delayTime);
```
Comment (matkatmusic) [inline main.cpp]: - [x] Pass them all to a single call of `ignoreUnused`, like you do in the functions below.

File: main.cpp [hunk 2]
```diff
@@ -81,72 +121,131 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    return{};
+}
 /*
  10)
  */
+bool running(bool start, bool stop)
+{
+    ignoreUnused(start, stop);
+    return{};
+}
```
Comment (matkatmusic) [inline main.cpp]: add space around your `{}`:  `return {};`
- [x] revise, improving the readability of the code.
- [x] `wide` is a reserved word, hence the reason it is blue here.  rename it to something else.

File: main.cpp [hunk 3]
```diff
@@ -81,72 +121,131 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
     ignoreUnused(carRented);
-    std::cout << "good to go!" << std::endl;
+    ignoreUnused(busDriving);
+    ignoreUnused(rendering);
+    ignoreUnused(phasing);
+    ignoreUnused(eqing);
+    ignoreUnused(limiting);
+    ignoreUnused(redZone);
+    ignoreUnused(stereoImage);
+    ignoreUnused(run);
```
Comment (matkatmusic) [inline main.cpp]: - [x] Pass them all to a single call of `ignoreUnused`

File: main.cpp [hunk 4]
```diff
@@ -81,72 +121,131 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    auto eqing = eq(34,87);
     //6)
-    
+    compressor(2,3);
     //7)
+    auto limiting = limiter(56,32);
     
     //8)
-    
+    auto redZone = clipping(52,84);
```
Comment (matkatmusic) [inline main.cpp]: add spaces around your `,` so it's easier to read.  you did this on lines 222 and 219 already. 
- [x] revise, improve readability.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Good.  most changes needed are cosmetic to improve readability.

File: main.cpp (hunk 1 above)
Comment (martin-mueller-xyz) [reply main.cpp]: ok. done.

File: main.cpp (hunk 2 above)
Comment (martin-mueller-xyz) [reply main.cpp]: add space around return {};
changed wide to spread

File: main.cpp (hunk 3 above)
Comment (martin-mueller-xyz) [reply main.cpp]: Passed all to a single call of ignoreUnused

Comment (MartinMueller-StudioMood) [reply main.cpp:248]: Passed all to a single call of ignoreUnused

File: main.cpp (hunk 4 above)
Comment (martin-mueller-xyz) [reply main.cpp]: added spaces around  ,

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**